docker/install-dynamic-plugins.py

```diff
@@ -167,8 +209,13 @@ def main():
             print('\n======= Skipping disabled dynamic plugin', package, flush=True)
             continue
 
-        if package.startswith('./'):
```

Comment:
Could you bring line 220 just before this line so that we log with the unchanged `package` name ?